sympy/core/tests/test_expr.py

```diff
@@ -1832,3 +1832,8 @@ def test_normal():
     x = symbols('x')
     e = Mul(S.Half, 1 + x, evaluate=False)
     assert e.normal() == e
+
+
+def test_expr():
+    x = symbols('x')
+    raises(TypeError, lambda: tan(x).series(x, 2, oo, "+"))
```

Comment:
There is now a merge conflict for these changes (presumably because someone else added to the end of this file).

Can you resolve that by moving this to test_series.py? Maybe there is already a test function there that you can add to. Otherwise a better name is needed than `test_expr` which is not very specific.

Reply:
Looks like @smichr resolved the merge conflicts